### PR TITLE
style(docs): integrate the stark layout in all showcase's pages

### PR DESCRIPTION
**Pull request by catlabs**
ISSUES CLOSED: #673

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/NationalBankBelgium/stark/blob/master/CONTRIBUTING.md#-commit-message-guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: #673

## What is the new behavior?
Stark layout is now applied in all pages of the showcase

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: showcase/src/app/home/home.component.html [hunk 1]
```diff
@@ -1,43 +1,100 @@
...
+<h1 class="mat-display-2" translate>Stark documentation</h1>
+
+<div fxLayout="column" fxLayout.gt-xs="row wrap" fxLayoutAlign.gt-md="space-between center" fxLayoutAlign.md="space-around center"
+ fxLayoutAlign.lt-md="center center" fxLayoutGap.lt-lg="20px">
+	<a href="https://stark.nbb.be/api-docs/stark-core/latest">
+		<mat-card class="stark-card">
 			<mat-card-header>
 				<mat-card-title>
-					<h1 class="documentation-title">Stark documentation</h1>
+					<h4 translate>Stark Core - Latest</h4>
```
Comment (christophercr) [inline showcase/src/app/home/home.component.html]: For the titles of the cards of this Home page, could we have them centered horizontally?

File: showcase/src/assets/translations/en.json [hunk 2]
```diff
@@ -5,18 +5,23 @@
...
-        "ROOT": "Root ancestor",
         "GRANDPARENT": "Grand parent",
+        "NO_INPUT": "Breadcrumb demonstration (without parameters)",
         "PARENT": "Parent",
-        "CHILD": "Child"
+        "ROOT": "Root ancestor",
+        "TITLE": "Stark breadcrumb"
+      },
+      "BUTTON": {
+        "TITLE": "Stark buttons"
```
[the remark this answers is not in the document]
Comment (catlabs) [reply showcase/src/assets/translations/en.json]: Well I don't think it is super relevant, title and sentences should be written with normal case, then we can css rule to override this, but at least we can always go back to the original version if we want.
I also considered uppercasing titles but they don't do it neither in Angular Material or in Material Design.

File: showcase/src/app/demo/date-range-picker/date-range-picker.component.html [hunk 3]
```diff
@@ -1,20 +1,13 @@
...
-			[startDate]="startDate" 
-			startDateLabel="From" 
-			[startMinDate]="minDate"
-			[startMaxDate]="maxDate" 
-			(dateRangeChanged)="onDateChanged($event)">
-		</stark-date-range-picker>
-	</div>
-    <mat-checkbox [(ngModel)]="isDisabled">isDisabled</mat-checkbox>
-</example-viewer>
+<h1 class="mat-display-3" translate>SHOWCASE.DEMO.DATE_PICKER.TITLE</h1>
```
[the remark this answers is not in the document]
Comment (catlabs) [reply showcase/src/app/demo/date-range-picker/date-range-picker.component.html]: Well spotted :-)

**Comment by catlabs** [comment]
Rebased and repushed

File: showcase/src/assets/translations/en.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (christophercr) [reply showcase/src/assets/translations/en.json]: Ok, let's keep it like this for now 😉

**Pull request merged**